tests/test_config_reader.py

```diff
@@ -318,7 +318,31 @@ def test_construct_stacks_constructs_stack(
                 "project_path": self.context.project_path,
                 "custom_key": "custom_value",
             },
-            config=ANY,
+            config={
+                "project_path": self.context.project_path,
+                "stack_group_path": "account/stack-group/region",
+                "profile": "account_profile",
+                "project_code": "account_project_code",
+                "region": "region_region",
+                "required_version": ">1.0",
+                "template_bucket_name": "stack_group_template_bucket_name",
+                "custom_key": "custom_value",
+                "dependencies": ["child/level", "top/level"],
+                "template": {"path": "path/to/template"},
+                "parameters": {"param1": "val1"},
+            },
+            vars={
+                "var": {},
+                "project_path": self.context.project_path,
+                "stack_group_path": "account/stack-group/region",
+                "profile": "account_profile",
+                "project_code": "account_project_code",
+                "region": "region_region",
+                "required_version": ">1.0",
+                "template_bucket_name": "stack_group_template_bucket_name",
+                "custom_key": "custom_value",
+                "dependencies": ["top/level"],
+            },
```

Comment:
Let's not do this. This is a lot of very particular keys and I don't see it getting updated easily. There's a better way, I think.